orchestration/dagster_orchestration/hca_orchestration/solids/copy_project/create_scratch_area.py

```diff
@@ -0,0 +1,15 @@
+from dagster import solid, ResourceDefinition
+from dagster.core.execution.context.compute import (
+    AbstractComputeExecutionContext,
+)
+
+
+@solid(
+    required_resource_keys={"snapshot_config", "gcs", "data_repo_client"}
+)
+def create_scratch_area(context: AbstractComputeExecutionContext):
```

Comment:
⛏️ This will need a return type for mypy to pass, `-> str:` should be fine for now.